client/v3/kubernetes/interface.go

```diff
@@ -0,0 +1,62 @@
+package kubernetes
+
+import (
+	"context"
+
+	"go.etcd.io/etcd/api/v3/mvccpb"
+	clientv3 "go.etcd.io/etcd/client/v3"
+)
+
+type Interface interface {
```

Comment:
Can the godoc for this interface be mostly copy-pasted from the existing interfaces and then augmented with k8s specifics?